### ISSUE TITLE
the variable value is incorrect when I use named request to get value of Long data type from response body

### ISSUE THREAD

**Issue by terryliulj** [Huachao/vscode-restclient#439]
- VSCode Version: 1.39.2
- OS Version: macos 10.14.6
- REST Client Version: 0.22.2

the variable value is incorrect when I use named request to get value of Long data type from response body.
For example:
@aid = {{getObj.response.body.$.data.aid}}
the actual value of aid is 3756818201733464064, but the value of @aid is 3756818201733464000

**Comment by terryliulj** [comment]
That is related to the length of the numeric type of js. May not be a bug.

**Comment by Huachao** [comment]
@terryliulj  you are right, since the number is larger than the maximum number `2 ^ 53 - 1`, and the native `JSON.parse` doesn't work as expected. I am still considering any other plans to solve it.

**Comment by tommck** [comment]
This is a very old bug that just bit me today.  No solution yet, @Huachao?  Any ideas? Might be able to request community help if there's a desired solution and not enough time to implement
